spec/lib/pre_assembly/digital_object_spec.rb

```diff
@@ -429,7 +422,7 @@ def add_object_files(extension:, num: 2, rel_path: '')
 
     describe 'book (rtl) structural metadata' do
       let(:cocina_type) { Cocina::Models::ObjectType.book }
-      let(:content_structure) { 'simple_book' }
+      let(:content_structure) { 'simple_book_rtl' }
 
```

Comment:
this is going to have to change a bit after https://github.com/sul-dlss/pre-assembly/pull/1453